src/transforms/transform-branch.ts

```diff
@@ -39,11 +40,9 @@ export const transformBranch = async (github: GitHubInstallationClient, webhookP
 	}
 
 	try {
-		const lastCommit = await getLastCommit(github, webhookPayload, issueKeys);
+		const lastCommit = await getLastCommit(gitHubInstallationClient, webhookPayload, issueKeys);
 		return {
-			id: repository.id.toString(),
-			name: repository.full_name,
-			url: repository.html_url,
+			... transformRepositoryDevInfoBulk(repository, gitHubInstallationClient.baseUrl),
```

Comment:
Drag-n-drop replacement (with exception of changing repository.name to repository.full_name - this is what we have in other places), not feature-flagging
<img width="1304" alt="image" src="https://user-images.githubusercontent.com/20631664/194985757-7f3dc9bf-707e-4fd9-8ec4-21099ac46206.png">
(on the left - removed lines, on the right - the mapping inside the function)